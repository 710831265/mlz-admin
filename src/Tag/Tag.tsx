import React, { useState, useContext } from 'react';
import { Tag as AntdTag } from 'antd';

const Tag = AntdTag;

export default Tag;
