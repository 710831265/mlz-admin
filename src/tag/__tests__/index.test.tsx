import React, { Component } from 'react';
import testMount from '../../../tests/testMount';
import testSnapshot from '../../../tests/testSnapshot';
import Tag from '..';

describe('ðŸ§ª Tag', () => {
  testMount(Tag);
  testSnapshot(Tag);
});
