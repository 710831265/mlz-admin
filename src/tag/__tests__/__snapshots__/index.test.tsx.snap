// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ðŸ§ª Tag snapshot test snapshot comparing 1`] = `"<span class=\\"ant-tag\\"></span>"`;
